src/Equinox.MessageDb/MessageDb.fs

```diff
@@ -180,85 +67,62 @@ module Read =
     let inline len (bytes: EventBody) = bytes.Length
     let private resolvedEventLen (x: ITimelineEvent<EventBody>) = len x.Data + len x.Meta
     let private resolvedEventBytes events = events |> Array.sumBy resolvedEventLen
-    let private loggedReadSlice reader streamName batchSize requiresLeader startPos batchIndex (log: ILogger) ct: Task<_> = task {
+    let private loggedReadSlice reader streamName batchSize requiresLeader startPos batchIndex ct: Task<_> = task {
         let act = Activity.Current
```

Comment:
any reason not to move down to first use?

Reply:
(but see comments later about maybe inlining it)

Reply:
Introduced a small helper for this